Workflows/owlbot.py

```diff
@@ -14,46 +14,35 @@
 
 """This script is used to synthesize generated parts of this library."""
 
-import synthtool as s
-import synthtool.gcp as gcp
 import logging
+from pathlib import Path
+import subprocess
 
-logging.basicConfig(level=logging.DEBUG)
-
-gapic = gcp.GAPICBazel()
-common = gcp.CommonTemplates()
+import synthtool as s
+from synthtool.languages import php
+from synthtool import _tracked_paths
 
-workflows_library = gapic.php_library(
-    service='workflows',
-    version='v1beta',
-    bazel_target='//google/cloud/workflows/v1beta:google-cloud-workflows-v1beta-php',
-)
+logging.basicConfig(level=logging.DEBUG)
 
-# copy all src including partial veneer classes
-s.move(workflows_library / 'src')
+src = Path(f"../{php.STAGING_DIR}/Workflows").resolve()
+dest = Path().resolve()
 
-# copy proto files to src also
-s.move(workflows_library / 'proto/src/Google/Cloud/Workflows', 'src/')
-s.move(workflows_library / 'tests/')
+# Added so that we can pass copy_excludes in the owlbot_main() call
+_tracked_paths.add(src)
 
-# copy GPBMetadata file to metadata
-s.move(workflows_library / 'proto/src/GPBMetadata/Google/Cloud/Workflows', 'metadata/')
+php.owlbot_main(src=src, dest=dest)
 
-executions_library = gapic.php_library(
-    service='workflows-executions',
-    version='v1beta',
-    bazel_target='//google/cloud/workflows/executions/v1beta:google-cloud-workflows-executions-v1beta-php',
-)
+executions_library = Path(f"../{php.STAGING_DIR}/Workflows/v1beta/Executions").resolve()
 
 # copy all src including partial veneer classes
-s.move(executions_library / 'src', 'src/Executions')
+s.move(executions_library / 'src', 'src/Executions', merge=php._merge)
```

Comment:
What is `php._merge`?

Reply:
That's a small python function for preserving copyright year.

Reply:
Could we add a comment or maybe set it to a variable to make this clear? maybe something like

```python
preserve_copyright_year = php._merge
```

Or just 

```python
# copy all src including partial veneer classes
# php._merge preserves the copyright year
s.move(executions_library / 'src', 'src/Executions', merge=php._merge)
```

Reply:
Good idea, done